tests/operators/test_local.py

```diff
@@ -462,7 +462,6 @@ def test_run_operator_dataset_inlets_and_outlets(caplog):
     assert test_operator.outlets == []
 
 
-# TODO: Add compatibility for Airflow 3. Issue: https://github.com/astronomer/astronomer-cosmos/issues/1704.
 @pytest.mark.skipif(version.parse(airflow_version).major == 3, reason="Test need to be updated for Airflow 3.0")
```

Comment:
Should we update the reason for skipping the test here?

Reply:
Updated in 2a113a5